testutils/g109_samples.go

```diff
@@ -20,7 +20,7 @@ func main() {
 	value := int32(bigValue)
 	fmt.Println(value)
 }
-`}, 2, gosec.NewConfig()},
+`}, 1, gosec.NewConfig()},
```

Comment:
ditto

Reply:
same reason as above.
1 issue of G109 and other of G115.